scripts/ci/tags.yaml

```diff
@@ -154,3 +154,35 @@ crypto:
     - zephyr/include/drivers/serial/
     - zephyr/modules/mbedtls/
     - modules/crypto/
+
+zigbee:
+  files:
```

Comment:
What about following files included in test-spec.yaml?
- nrf/applications/zigbee_weather_station
- nrf/samples/nrf5340/multiprotocol_rpmsg

Reply:
Added.